.github/workflows/dependencies.yml

```diff
@@ -31,6 +31,9 @@ jobs:
   update:
     name: Update
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
```

Comment:
Why do we need the both permissions set to write?

Reply:
I just removed the unneeded permissions as it appears to work without it https://github.com/KolbyML/trin/pull/6

I added it as apparently in certain conditions it can be required it looks like

https://github.com/peter-evans/create-pull-request?tab=readme-ov-file#token
![image](https://github.com/user-attachments/assets/85f123b1-d968-4881-b9e1-4e7f6e1154a1)